examples/helloapache/README.md

```diff
@@ -1,8 +1,10 @@
+# helloapache
+
 This is an atomic application based on the nulecule specification. Kubernetes is currently the only supported provider. You'll need to run this from a workstation that has the atomic CLI and kubectl client that can connect to a kubernetes master.
 
-It's a single pod based on the centos/httpd image, but you can use your own.
+It's a single container application based on the centos/httpd image, but you can use your own.
 
```

Comment:
The problem is that we support more than kubernetes in the example and hte use of the word pod implies to me that it is always a kubernetes example.  We can expand the sentence to state that when the example is run with a kubernetes provider it creates a single pod, if you wish.  What do you think?